### PR TITLE
Update My-PR-Contribution-Practice.md

### PR DESCRIPTION
**Pull request by Fuglydami**
#ISSUE-NO

### CODE REVIEW

**Review by chryzcode — CHANGES_REQUESTED** [review]
Please put your name in an alphabetical order. Thanks

File: My-PR-Contribution-Practice.md [hunk 1]
```diff
@@ -46,3 +46,7 @@ ________________________________________________________________________________
 
 - [Udeagbala Anthony](https://github.com/izudada)
   <p>Bio: A growing python developer, more interested in building web applications with python. </p>
+
+- [Damilare Oyedeji](https://github.com/fuglydami)
```
Comment (Fuglydami) [inline My-PR-Contribution-Practice.md:50]: Thought this is already in alphabetical order. No? if not, please how do i make the changes, i'm new to open source.

Comment (chryzcode) [reply My-PR-Contribution-Practice.md:50]: Okay, your name starts with the word **D**amilare.
Alphabetically just means that put your name in the order of ABC.
- You are putting your name behind **U**deagbala which is not meant to be so
- Your name should be up, after any name that starts with the word **C** if there is none, then check **B** if there is none, there is **A** there.

**Review by amjadnoor — CHANGES_REQUESTED** [review]
@Fuglydami If you don't understand what to make changes that we are talking about then place your name and Bio between Amjad Noor and Franklin Ohaegbulam.

**Comment by frankiefab100** [comment]
@Fuglydami if you still don't understand, do well to comment.

**Comment by chryzcode** [comment]
Ok, I would merge it and help him fix it!

**Review by chryzcode — APPROVED** [review]

**Pull request merged**